src/EventLoop.php

```diff
@@ -52,13 +69,17 @@ public function promiseRejected(\Throwable $throwable): Promise;
 
     /**
      * Creates a promise that will be resolved to null in a future tick of the event loop.
+     *
+     * @return Promise<void|null>
      */
     public function idle(): Promise;
 
     /**
      * Creates a promise that will be resolved to null after a fixed time.
      * ⚠️ The actual measured delay can be greater if your event loop is busy!
      * It also can be a little smaller, depending on your event loop (or system clock) accuracy.
+     *
+     * @return Promise<void|null>
```

Comment:
Same question here :)
```suggestion
     * @return Promise<null>
```

Reply:
Tornado implementation was using a generator which didn't return anything, fixed the implementation now :+1: